modules/Broadcast/BroadcastStory.tsx

```diff
@@ -0,0 +1,50 @@
+'use client';
+
+import type { Story } from '@prezly/sdk';
+import type { ReactNode } from 'react';
+import { createContext, useCallback, useContext, useEffect, useMemo, useState } from 'react';
+
+type Revoke = () => void;
+
+interface Context {
+    story: Story | null;
+    broadcast: (story: Story) => Revoke;
+}
+
+const context = createContext<Context>({
+    story: null,
+    broadcast() {
+        throw new Error(
+            'This functionality requires `BroadcastStoryProvider` mounted up the components tree.',
+        );
+    },
+});
+
+export function BroadcastStoryProvider(props: { children: ReactNode }) {
```

Comment:
Taking into consideration we are only interested in the story's UUID, do you think it would be better to only broadcast the UUID instead of the whole object? 🤔

Reply:
I wouldn't, as long as it doesn't hurt which in this case it seems it doesn't.

Kinda reminds me of how we pass props to components. I realized the pattern of passing the whole entity is just cleaner, less verbose and more maintainable in the long run.

Reply:
Yeah, I think it's okay too. The object should have a stable reference as well thanks to `useMemo` in case we need to use it as a dependency somewhere.